cpp/oneapi/dal/algo/pca/backend/cpu/finalize_train_kernel_svd.cpp

```diff
@@ -65,7 +65,12 @@ static train_result<Task> call_daal_kernel_finalize_train(const context_cpu& ctx
             interop::copy_to_daal_homogen_table<Float>(input.get_auxiliary_table(i));
         decomposeCollection->push_back(daal_crossproduct);
     }
+
     daal_pca::internal::InputDataType dtype = daal_pca::internal::nonNormalizedDataset;
+    //TODO: investigate opportunity to change default behavior
+    if (desc.is_scaled() == true && desc.is_mean_centered() == true) {
```

Comment:
Once again monstrous Boolean conditions

Reply:
Furthermore - it is not a consistent naming of member functions. We typically implement them in a getter/setter fashion. So that the more aligned naming for this function would be "get_do_scaling"/"set_do_scaling"